exercises/concept/cars-assemble/.meta/design.md

```diff
@@ -0,0 +1,28 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student about conditionals in R. It provides additional practice after `vehicle-purchase`.
+
+## Learning objectives
+
+- Know that the relational operators are `==`, `!=`, `>=`, `<=`.
+- Know the syntax of a block if-else.
+- Know about the `ifelse()` function.
```

Comment:
As before, remove `ifelse()` here.

Reply:
I'll certainly make these changes. For practical reasons, probably tomorrow.